fe/fe-core/src/main/java/org/apache/doris/nereids/NereidsPlanner.java

```diff
@@ -288,6 +296,18 @@ private Plan planWithoutLock(
         return physicalPlan;
     }
 
+    private List<LogicalOlapScan> getAllOlapScans(Plan plan) {
```

Comment:
could replace by `plan.collect(LogicalOlapScan.class::isInstance)`

Reply:
the meaning of -1 is explained in comments for getOlapTableRowCount()